assets/javascripts/initializers/discourse-calendar.js.es6

```diff
@@ -298,6 +307,35 @@ function initializeDiscourseCalendar(api) {
       }
     });
   }
+
+  function _getTimeZone($calendar, currentUser) {
+    let defaultTimezone = $calendar.attr("data-calendar-default-timezone");
+    const isValidDefaultTimezone = !!moment.tz.zone(defaultTimezone);
+    if (!isValidDefaultTimezone) {
+      defaultTimezone = null;
+    }
+
+    const currentUserTimezone = currentUser ? currentUser.timezone : null;
+    return defaultTimezone || currentUserTimezone || moment.tz.guess();
+  }
+
+  function _setupTimezonePicker(calendar, timeZone) {
+    let $timezonePicker = $(".discourse-calendar-timezone-picker");
+
+    if ($timezonePicker.length) {
+      $timezonePicker.on("change", function(e) {
```

Comment:
this is very minor nitpick, but I prefer `event` over `e`, easier to grep for event than e